bzt/resources/selenium_taurus_extras.py

```diff
@@ -0,0 +1,45 @@
+"""
+Copyright 2018 BlazeMeter Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from string import Template as StrTemplate
+
+
+class Apply(StrTemplate):
+
+    def __init__(self, template):
+        super(Apply, self).__init__(template)
+        self.variables = {}
+
+    def __repr__(self):
+        return repr(self.safe_substitute(self.variables))
+
+    def __str__(self):
+        return self.safe_substitute(self.variables)
+
+
+class Template:
+
+    def __init__(self, variables=None):
+        if dict:
```

Comment:
This should probably have been `if variables` or `if variables is not None`.

Reply:
ok, already solved (it was not right that it was optional)

645393f3219888cfe676747ce62ec54723ba3826